src/sql/src/plan/statement/ddl.rs

```diff
@@ -375,6 +383,80 @@ generate_extracted_config!(
     (TextColumns, Vec::<UnresolvedItemName>, Default(vec![]))
 );
 
+pub fn plan_create_webhook_source(
+    scx: &StatementContext,
+    stmt: CreateWebhookSourceStatement<Aug>,
+) -> Result<Plan, PlanError> {
+    // Make sure the LaunchDarkly flag is enabled.
+    scx.require_feature_flag(&vars::ENABLE_WEBHOOK_SOURCES)?;
+
+    let create_sql =
+        normalize::create_statement(scx, Statement::CreateWebhookSource(stmt.clone()))?;
+
+    let CreateWebhookSourceStatement {
+        name,
+        if_not_exists,
+        body_format,
+        include_headers,
+    } = stmt;
+
+    let body_scalar_type = match body_format {
+        Format::Bytes => ScalarType::Bytes,
+        Format::Json => ScalarType::Jsonb,
+        Format::Text => ScalarType::String,
+        // TODO(parkmycar): Make an issue to support more types, or change this to NeverSupported.
+        ty => {
+            return Err(PlanError::Unsupported {
+                feature: format!("{ty} is not a valid BODY FORMAT for a WEBHOOK source"),
```

Comment:
Oh yeah, like I was saying in the parser reivew, you're already erroring here so you can be more permissive in the parser

Reply:
Commented above in more detail, but I wanted to be more restrictive in the parser for better error messages. Let me know if you want me to change this 🙂

Reply:
This is good--ty